src/manifest.json

```diff
@@ -31,13 +31,16 @@
                 "mousetrap.js",
                 "mousetrap-global-bind.js",
                 "extension.js",
+                "searchEngines.js",
                 "main.js"
             ],
             "css": [
                 "search_results.css"
             ],
             "run_at": "document_end",
             "matches": [
+                "*://www.startpage.com/sp/search*",
+                "*://www.startpage.com/do/dsearch*",
```

Comment:
I removed startpage permission and added a checkbox to add them.
They are added programmatically thanks to [this](https://github.com/fregante/webext-dynamic-content-scripts). Since it use [`browser.contentScripts.register()`](https://github.com/fregante/content-scripts-register-polyfill) to register scripts on new domains added via `permission.request()`, the manifest will stay the same and no new permisisons will be required when adding a new search engine.